Swift/ConflictResolver.swift

```diff
@@ -20,9 +20,38 @@
 import Foundation
 
 /// Conflict Resolver protocol
-public protocol ConflictResolver {
+public protocol ConflictResolverProtocol {
     
     /// The callback resolve method, if conflict occurs.
     func resolve(conflict: Conflict) -> Document?
     
 }
+
+
+/// Conflict Resolution class
+public class ConflictResolver {
+    
+    /// The default conflict resolver used by the replicator.
+    public static var `default`: ConflictResolver {
+        return DefaultResolver()
```

Comment:
Can we do public static let `default` so that there will be only one single instance of the DefaultResolver?